src/services/scenario/ScenarioService.js

```diff
@@ -21,10 +22,33 @@ async function setValidationStatus(organizationId, workspaceId, scenarioId, vali
   return Api.Scenarios.updateScenario(organizationId, workspaceId, scenarioId, data);
 }
 
+async function updateSecurity(organizationId, workspaceId, scenarioId, currentSecurity, newSecurity) {
+  const setDefaultSecurity = async (newRole) =>
+    Api.Scenarios.setScenarioDefaultSecurity(organizationId, workspaceId, scenarioId, { role: newRole });
+  const addAccess = async (newEntry) =>
+    Api.Scenarios.addScenarioAccessControl(organizationId, workspaceId, scenarioId, newEntry);
+  const updateAccess = async (userIdToUpdate, newRole) =>
+    Api.Scenarios.updateScenarioAccessControl(organizationId, workspaceId, scenarioId, userIdToUpdate, {
+      role: newRole,
+    });
+  const removeAccess = async (userIdToRemove) =>
+    Api.Scenarios.removeScenarioAccessControl(organizationId, workspaceId, scenarioId, userIdToRemove);
```

Comment:
in this function, there are four `async` without any `await`, I'm a bit confused :thinking:

Reply:
The four `async` functions declared between lines 26 and 35 are _callback functions_ that will be called by `SecurityUtils.updateResourceSecurity` (i.e. the 4 `await`s are in `SecurityUtils.updateResourceSecurity` :smile: ).